machine/alphanetworks/alphanetworks_snh61d2_320f/kernel/config

```diff
@@ -0,0 +1,9 @@
+CONFIG_IGB=y
+CONFIG_I2C_I801=y
+CONFIG_X86_X2APIC=y
+CONFIG_USB_XHCI_HCD=y
+CONFIG_EFI=y
+CONFIG_EFIVAR_FS=y
```

Comment:
Same thing here -- no need to set these EFI config options.  They are already enabled in the base config.